cocos/core/math/quat.ts

```diff
@@ -224,6 +237,7 @@ export class Quat extends ValueType {
     }
 
     /**
+     * @en Space around the world at a given axis of rotation quaternion
```

Comment:
Sets the out quaternion to represent a radian rotation around a given rotation axis in world space